### PR TITLE
chore(deps-dev): update rector/rector requirement from 0.18.13 to 0.19.2

### PR DESCRIPTION
**Pull request by datamweb**
**Description**
This PR will upgrade Reactor to version 0.19.2.

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

**Comment by samsonasik** [comment]
Rector already version 0.19.2

File: composer.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
         "mockery/mockery": "^1.0",
         "phpstan/extension-installer": "^1.3",
         "phpstan/phpstan-strict-rules": "^1.5",
-        "rector/rector": "0.18.13"
+        "rector/rector": "0.19.0"
```
Comment (samsonasik) [inline composer.json]:
```suggestion
        "rector/rector": "0.19.2"
```

**Comment by samsonasik** [comment]
You can replace `Rector\Core\` with `Rector\` in `rector.php`

**Review by samsonasik — APPROVED** [review]

**Review by kenjis — APPROVED** [review]

**Comment by datamweb** [comment]
@samsonasik thank you!

**Pull request merged**